presto-native-execution/presto_cpp/main/Announcer.h

```diff
@@ -57,6 +59,8 @@ class Announcer {
   folly::EventBaseThread eventBaseThread_;
   const std::string clientCertAndKeyPath_;
   const std::string ciphers_;
+  uint64_t failedAttempts_{0};
+  const double jitterParam{0.1};
```

Comment:
NYC: could rearrange the members by putting the const members first? Thanks!

Can you comment on jitterParam_?
s/jitterParam/jitterParam_/